src/JsonDecodingException.php

```diff
@@ -2,7 +2,5 @@
 
 namespace League\JsonReference;
 
-final class JsonDecodingException extends \RuntimeException
-{
-
-}
+// Legacy
+class_alias('DecodingException', 'JsonDecodingException');
```

Comment:
I would rather make this `final class JsonDecodingException extends DecodingException`.

Reply:
I cant do this because DecodingException is final